src/services/validateZendeskRequest.spec.ts

```diff
@@ -0,0 +1,412 @@
+import { validateZendeskRequest } from './validateZendeskRequest'
+import { IdentifierTypes } from '../types/dataRequestParams'
+describe('validateZendeskRequest', () => {
```

Comment:
A very corner case, but could be expected since we can't remove the attachment field. If the form has an attachment, would that be submitted along, and if so, do we just ignore it or send Zendesk some message? Not sure if the Zendesk macros would have already discarded that.

Reply:
Don't know much about the implementation, but I assume what you see in Zendesk is not what is sent via the webhook. We are in control of what we send to the API, so we don't have to send every field from Zendesk.

Maybe @mabon-gds can confirm in Andrew's absence?

Reply:
I think we should be able to remove the attachment field as I remember seeing that a template. I'll investigate this and action it separately.

Reply:
I can't see an easy way to remove the field as I previously commented but back to the original concern. Andy is right here, we only send the fields we're concerned with so even if someone attaches something to the form, by design, we're not including it in the webhook. The same goes for Subject and Description fields both of which are system fields. You can see the details [here](https://govuk1620731396.zendesk.com/admin/objects-rules/rules/triggers/5711827450012) of what's sent here.

Reply:
Thanks for clarifying.